protobuf/source_reflection_test.go

```diff
@@ -37,6 +38,15 @@ func TestReflection(t *testing.T) {
 	assert.Nil(t, result)
 }
 
+func TestReflectionWithProtocolInPeer(t *testing.T) {
+	source, err := NewDescriptorProviderReflection(ReflectionArgs{
+		Timeout: time.Second,
+		Peers:   []string{"grpc://127.0.0.1:12345"},
+	})
+	assert.Contains(t, err.Error(), "peer contains scheme")
```

Comment:
do a `require.Error` before the `assert.Contains` so we don't panic in case it returns no error